triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceAppConf.scala

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger
+
+import akka.http.scaladsl.model.Uri
+import com.daml.dbutils.JdbcConfig
+import com.daml.lf.speedy.Compiler
+import com.daml.platform.services.time.TimeProviderType
+import pureconfig.{ConfigReader, ConvertHelpers}
+import com.daml.auth.middleware.api.{Client => AuthClient}
+import pureconfig.error.FailureReason
+import pureconfig.generic.semiauto.deriveReader
+
+import java.nio.file.Path
+import java.time.Duration
+import scala.concurrent.duration.FiniteDuration
+
+private[trigger] object LedgerApiConfig {
+  lazy implicit val ledgerApiCfgReader: ConfigReader[LedgerApiConfig] =
```

Comment:
Why is this `lazy`? `object`s are already initialized lazily, would that be enough?

Reply:
it is indeed need not be `lazy`, I think I ran into an error at some point along the lines of 
```
could not find Lazy implicit value of type pureconfig.generic.DerivedConfigReader
```
and albeit in a dumb way proceeded to make things lazy, its not required and I will cleanup

Reply:
For future reference, `val` members of `object`s are initialized eagerly when the object is referenced, so the `object`'s laziness does not help you in the use cases that call for `lazy val`s:

```scala
scala> object Foo { val bar = {println("bar hi"); "a"}; val baz = "b" }
object Foo

scala> Foo
bar hi
val res0: Foo.type = Foo$@383cdd4d
```